impedance/models/circuits/elements.py

```diff
@@ -1,6 +1,21 @@
-import numpy as np
 import cmath
 
+import numpy as np
+
+initial_state = globals().copy()
+non_element_functions = ['element_metadata',
+                         'initial_state',
+                         'non_element_functions',
+                         's',
+                         'p',
+                         'K',
```

Comment:
`K` is also an element that is used for the LinKK method, so I think it can also be added here too